src/models/MySqlModel.ts

```diff
@@ -110,7 +112,7 @@ export class MySqlModel {
   static async query(
     apolloContext: MyContext,
     sqlStatement: string,
-    values: string[] = [],
+    values: MixedArray<string | boolean> = [],
```

Comment:
I added this because the /signup page POSTs a mix of strings and a boolean for "acceptedTerms"